docs/guides/wipe-environment.rst

```diff
@@ -0,0 +1,28 @@
+Wiping a Build/Environment
+==========================
+
+Sometimes it happen that your Builds start failing because the
+environment where the documentation is created is stale or
+broken. This could happen for a couple of different reasons like `pip`
+not upgrading a package properly or a corrupted cached Python package,
+for example.
+
+In any of these cases (and many others), the solution could be just
+wiping out the existing build/environment files and allow Read the
```

Comment:
`build/environment` is weird. I'd just choose one and stick with it, perhaps `build environment`